sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -3339,10 +3340,10 @@ public partial class RangeFacetResult<T> where T : struct
         public T? From { get { throw null; } }
         public T? To { get { throw null; } }
     }
-    public partial class RawVector : Azure.Search.Documents.Models.SearchQueryVector
+    public partial class RawVectorQuery : Azure.Search.Documents.Models.VectorQuery
     {
-        public RawVector() { }
-        public System.Collections.Generic.IList<float> Vector { get { throw null; } }
+        public RawVectorQuery() { }
+        public System.Collections.Generic.IReadOnlyList<float> Vector { get { throw null; } set { } }
```

Comment:
Collection properties should not be settable. It's in the guidelines.

Reply:
This is a specific case for embedding where they are immutable. We've already discussed this case [here](https://github.com/Azure/azure-sdk-for-net/pull/36109#discussion_r1187853918), and @tg-msft has approved this change.

Reply:
This is a specific case for embeddings. We've already discussed this case [here](https://github.com/Azure/azure-sdk-for-net/pull/36109#discussion_r1187853918), and @tg-msft has approved this change.